addresses/1.json

```diff
@@ -2,6 +2,7 @@
   "DROP_METADATA_RENDERER": "0x5914d9a241008b9f02f22811bf3a77e02b84d226",
   "EDITION_METADATA_RENDERER": "0x192ce8267cbab9c3c477d61e85d7f0c5fe3b46af",
   "ERC721DROP_IMPL": "0x8237F421357F87a23ed0CFf3a5586172F210A21B",
+  "ERC721DROP_IMPL_VERSION": 14,
```

Comment:
this would be easier to parse as each deployment having a key, with an address and a version, something like:

```json
ERC721DROP_IMPL: {
    address: "0x82..."
    version: 14
}
```

Reply:
also...please check if this change breaks:
`node script/copy-deployed-contracts.mjs`

Reply:
and also can we have the deployment script set these versions?

Reply:
Added the solidity to fix the json import script.

Reply:
We may want to revamp these entire files but having a flat structure is easier to work with in forge. They did add attribute getters which i think solve this but I would rather revamp these entire files rather than just change this field. Can look into a stacked diff.